docs/app/docs/ide_configuration/direnv.md

```diff
@@ -20,16 +20,17 @@ If you see any errors when activating your `.envrc` file, you will need to run `
 
 #### New Project
 
-If you have direnv installed, Devbox will generate an .envrc file when you run `devbox init` and enables it by running `direnv allow` in the background:
+If you have direnv installed, Devbox will generate an .envrc file when you run `devbox generate direnv` and enables it by running `direnv allow` in the background:
 
 ```bash
-➜  devbox init
+➜  devbox generate direnv
+Success: generated .envrc file
+Success: ran `direnv allow`
 direnv: loading ~/src/devbox/docs/.envrc
 direnv: using devbox
 ```
 
-This will generate a `.envrc` file in your root directory along with your `devbox.json`, you can run `direnv allow` so that your shell will activate whenever you navigate to the directory 
-or `direnv revoke` to stop.
+This will generate a `.envrc` file in your project directory along with your `devbox.json`, if you make changes to `.envrc` file, you can run `direnv allow` so that your shell will activate whenever you navigate to the directory or `direnv revoke` to stop. Any changes to `devbox.json` will automatically trigger direnv to re-setup the environment.
```

Comment:
```suggestion
This will generate a `.envrc` file in your project directory that contains `devbox.json`. Run `direnv allow` to activate your shell upon directory navigation. Run `direnv revoke` to stop. Changes to `devbox.json` automatically trigger direnv to reset the environment.
```